b92.html

```diff
@@ -17,7 +17,7 @@
 
 <body>
          
-
+		 
```

Comment:
You need to do two things:
1) Eliminate so many unnecessary new lines which you have
2) Add indentation, for example:
```
<body>
 <div>
  <h2>B92</h2>
 </div>
</body>
```